lib/dbservice_web/controllers/student_controller.ex

```diff
@@ -32,7 +32,9 @@ defmodule DbserviceWeb.StudentController do
   end
 
   def index(conn, params) do
-    param = Enum.map(params, fn {key, value} -> {String.to_existing_atom(key), value} end)
+    query_params = Map.delete(params, "is_filtered")
```

Comment:
`is_filtered` looks like you are sending a filtered list of students. But instead you are not having any filters applied but rather have reduced columns. I would suggest using a different name for this param if you want to get reduced set of columns.

Maybe something like `response_size=compact` will work?